.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
If we can't get the most common error without a plugin, let's not check in config + deps on eslint.